### PR TITLE
Claire fixes things

### PR DESCRIPTION
**Pull request by MissXera**
- added jumpthrow bind
- r_drawtracers_firstperson should always be 1
- added r8 to weapon alias list
- host_writeconfig fix

### CODE REVIEW

File: autoexec.cfg [hunk 1]
```diff
@@ -36,7 +36,6 @@ hud_scaling .60
 
 // Miscellaneous
 cl_show_observer_crosshair 0
-cl_show_observer_crosshair 0
```
Comment (MissXera) [inline autoexec.cfg:39]: redundant

File: autoexec.cfg [hunk 2]
```diff
@@ -69,7 +67,7 @@ mat_queue_mode "-1" // auto detect multi-core rendering
 fps_max "160"
 fps_max_menu "145"
 r_dynamic "0"
-r_drawtracers_firstperson "0"
+r_drawtracers_firstperson "1"
```
Comment (MissXera) [inline autoexec.cfg:70]: should always be 1 after recent updates, minor performance impact

File: autoexec.cfg [hunk 3]
```diff
@@ -142,6 +140,7 @@ alias "glock" "give weapon_glock"
 alias "p2000" "give weapon_hkp2000"
 alias "P250" "give weapon_P250"
 alias "usp" "give weapon_usp_silencer"
+alias "r8" "give weapon_revolver"
```
Comment (MissXera) [inline autoexec.cfg:143]: cant practice properly without r8

File: autoexec.cfg [hunk 4]
```diff
@@ -201,6 +200,10 @@ bind x "use weapon_smokegrenade"
 bind c "use weapon_hegrenade"
 bind v "use weapon_molotov;use weapon_incgrenade"
 
+alias "+jumpthrow" "+jump;-attack"
+alias "-jumpthrow" "-jump"
+bind "UNKNOWN" "+jumpthrow"
+
```
Comment (MissXera) [inline autoexec.cfg:206]: mandatory missing? jumpthrow bind

**Review by anaisthatyou — APPROVED** [review]

**Review by anaisthatyou — APPROVED** [review]

**Pull request merged**